src/test/java/com/github/dockerjava/netty/exec/LoadImageCmdExecTest.java

```diff
@@ -0,0 +1,73 @@
+package com.github.dockerjava.netty.exec;
+
+import com.github.dockerjava.api.model.Image;
+
+import com.github.dockerjava.netty.AbstractNettyDockerClientTest;
+import org.testng.ITestResult;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.io.InputStream;
+import java.lang.reflect.Method;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+@Test(groups = "integration")
+public class LoadImageCmdExecTest extends AbstractNettyDockerClientTest {
+
+    private String expectedImageId;
+
+    @BeforeTest
+    public void beforeTest() throws Exception {
+        super.beforeTest();
+    }
+
+    @AfterTest
+    public void afterTest() {
+        super.afterTest();
+    }
+
+    @BeforeMethod
+    public void beforeMethod(Method method) {
+        super.beforeMethod(method);
+        expectedImageId = "sha256:5d195fbfc7b1ee5cef048b156c2eb9a6e7f00db14420b65d99d0f74a24235764";
+        if (findImageWithId(expectedImageId, dockerClient.listImagesCmd().exec()) != null) {
+            dockerClient.removeImageCmd(expectedImageId).exec();
+        }
+    }
+
+    @AfterMethod
+    public void afterMethod(ITestResult result) {
+        dockerClient.removeImageCmd(expectedImageId).exec();
+        super.afterMethod(result);
+    }
+
+    @Test
+    public void loadImageFromTar() throws Exception {
+        final Path imageTarFile = Paths.get("src/test/resources/testLoadImageFromTar/image.tar");
```

Comment:
Then you can make public getter but place it in one place. The issue with placing just in resources that it will be like a trash can and after test deletions/refactorings they will be forgotten. 
@leonsabr will search for examples bit later.